.circleci/config.yml

```diff
@@ -452,6 +452,125 @@ jobs:
+            git clone --depth=1 https://github.com/RandomityGuy/hxDatachannel
+            cd hashlink
+            ln -s ~/deps/hxDatachannel/cpp libs/datachannel
+            echo -e "\nadd_subdirectory(datachannel)" >> libs/CMakeLists.txt
+            cmake -S. -Bbuild -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DWITH_SQLITE=OFF -DBUILD_TESTING=OFF -DHASHLINK_INCLUDE_DIR="~/deps/hashlink/src" -DHASHLINK_LIBRARY_DIR="/usr/local/lib/"
+            cmake --build build --config Release -j$NPROC
+            sudo cmake --install build
+
+      - run:
+          name: Install haxe dependencies
+          command: |
+            export PATH=~/haxe/:"$PATH"
+            export HAXE_STD_PATH=~/haxe/std
+            export PATH=~/neko/:"$PATH"
+            export LD_LIBRARY_PATH=~/neko
+            haxelib dev hashlink ~/deps/hashlink/other/haxelib
+            haxelib git heaps https://github.com/RandomityGuy/heaps
+            haxelib dev hlopenal ~/deps/hashlink/libs/openal
+            haxelib dev hlsdl ~/deps/hashlink/libs/sdl
+            haxelib dev datachannel ~/deps/hxDatachannel
+            haxelib install colyseus-websocket
+
+      - save_cache:
+          key: mbhaxe-deps-v1
+          paths:
+            - /usr/local/lib/datachannel.hdll
+            - /usr/local/lib/fmt.hdll
+            - /usr/local/lib/openal.hdll
+            - /usr/local/lib/sdl.hdll
+            - /usr/local/lib/ssl.hdll
+            - /usr/local/lib/ui.hdll
+            - /usr/local/lib/uv.hdll
+
+      - run:
+          name: Compile MBHaxe
+          command: |
+            export PATH=~/haxe/:"$PATH"
+            export HAXE_STD_PATH=~/haxe/std
+            export PATH=~/neko/:"$PATH"
+            export LD_LIBRARY_PATH=~/neko
+            cd ~/MBHaxe
+            haxe compile.hxml
+            haxe compile-linux.hxml
+            cd native
+            cp ~/deps/hashlink/src/hlc_main.c .
+            gcc -o marblegame -O2 -I . -L /usr/local/lib marblegame.c /usr/local/lib/{ui.hdll,openal.hdll,fmt.hdll,sdl.hdll,uv.hdll,ssl.hdll,datachannel.hdll} -lSDL2 -lhl -lm -luv
+            strip marblegame
+
+      - run:
+          name: Package Bundle
+          command: |
+            mkdir -p ~/MBHaxe-Ultra-Linux
+            cd ~/MBHaxe-Ultra-Linux
+            cp ~/MBHaxe/marblegame.hl .
+            cp ~/MBHaxe/native/marblegame .
+            cp ~/MBHaxe/linux-dist/* .
+            cp -r ~/MBHaxe/data data
+            cp /usr/local/lib/{{fmt,openal,sdl,ssl,ui,uv,datachannel}.hdll,libhl.so.1.13.0} .
+            cp /usr/local/bin/hl .
+            ln -s libhl.so.1.13.0 libhl.so.1
+            ln -s libhl.so.1 libhl.so
+            cd ..
+            tar -czvf MBHaxe-Ultra-Linux.tar.gz MBHaxe-Ultra-Linux
+
+      #- run:
+      #    name: Upload to Artifact Storage
+      #    command: |
+      #      scp -o StrictHostKeyChecking=no -i $KEYPATH -P $PORT ~/MBHaxe/macos-dist/MBHaxe-Ultra-Mac.dmg $REMOTEDIR/MBHaxe-Ultra-Mac.dmg
```

Comment:
Presumably this is some ssh server you are running locally? `store_artifacts` below worked for me. Might want to uncomment this block before merge to match the others

Reply:
Yeah please uncomment this.